utils/keychain/helpers.ts

```diff
@@ -132,12 +132,12 @@ export const deleteAccountEncryptionKey = (account: string) =>
 export const getDbEncryptionKey = async () => {
   const existingKey = await getSecureItemAsync("LIBXMTP_DB_ENCRYPTION_KEY");
   if (existingKey) {
-    return Buffer.from(existingKey, "base64");
+    return new Uint8Array(Buffer.from(existingKey, "base64"));
```

Comment:
Type checks were working, but looking at this file locally there were issues, so fixing that up in this